testing/demo/pkg/transfer/consts.go

```diff
@@ -8,9 +8,9 @@ import (
 
 var (
 	// transferAmount is the amount of tokens to transfer.
-	transferAmount = math.NewInt(100)
+	transferAmount = math.NewInt(1e5)
 	// transferBackAmount is the amount of tokens to transfer back.
-	transferBackAmount = big.NewInt(1)
+	transferBackAmount = big.NewInt(1e5)
```

Comment:
Can we send slightly less back so that we can invoke `make transfer-back` multiple times? Currently if we send it all back, then we have to restart the demo from the top

```suggestion
	transferBackAmount = big.NewInt(10_000)
```